### ISSUE TITLE
About [2,3]- fold cross validation scores (Ch-5, page 257, paragraph below Out[6])

### ISSUE THREAD

**Issue by gaussEuler** [amueller/introduction_to_ml_with_python#57]
Respected Dr. Muller,
Sorry for asking so many questions - I hope you wont mind.
This question pertains to the paragraph right below Out[6] on page 257. There you state (and it makes sense) that for k=3, the accuracy on the IRIS dataset is 0. 
However, when I do it, I do not get 0. Not only that, I get the score which is almost equal to that for k=5!

That is, running the following command:
from sklearn.datasets import load_iris
iris = load_iris()

from sklearn.linear_model import LogisticRegression
logreg = LogisticRegression()

from sklearn.model_selection import cross_val_score

for cv in [2, 3, 5, 10, 15, 20]:
    scores = cross_val_score(logreg, iris.data, iris.target, cv=cv)
    print('\n{ }-fold Cross-validation scores: { }'.format(cv, scores))    
    print('AVERAGE { }-fold Cross-validation scores: {:.2f}\n'.format(cv, scores.mean()))
    print('---------------------------------------------') 

I get the scores that are almost equal to each other. Why?
If my question has validity, and if you have time, then please help me with this. 

Thank you,
Sincerely,
Nikhilesh

**Comment by amueller** [comment]
by default sklearn does stratified cross validation, which resolves the issue, as explained in the next paragraph

**Comment by gaussEuler** [comment]
Thank you Dr. Muller, for the reply. I am sorry to waste your time - I should have been more careful and properly read the material before contacting you.
